src/xdpinstaller/xdpinstaller.wixproj

```diff
@@ -34,7 +34,7 @@
       <Project>{921ea48b-3d7b-4e5c-892b-6f72f0852714}</Project>
     </ProjectReference>
   </ItemGroup>
-  <PropertyGroup Condition=" '$(IsAdmin)' == 'False' ">
+  <PropertyGroup Condition=" '$(IsAdmin)' == 'False' OR '$(UndockedOfficial)' == 'true' ">
```

Comment:
i don't think this is what you want. you want onebranch (and its github validation) to validate the MSI, which requires running as admin, and both github and onebranch do run as admin AFAIK.